bgpd/bgp_attr.c

```diff
@@ -4117,9 +4117,12 @@ bgp_size_t bgp_packet_attribute(struct bgp *bgp, struct peer *peer,
 			stream_putc(s, BGP_ATTR_FLAG_OPTIONAL
 					       | BGP_ATTR_FLAG_TRANS);
 			stream_putc(s, BGP_ATTR_PREFIX_SID);
-			stream_putc(s, 24);     /* tlv len */
+			stream_putc(s, 28); /* tlv len */
 			stream_putc(s, BGP_PREFIX_SID_SRV6_L3_SERVICE);
-			stream_putw(s, 21);     /* sub-tlv len */
+			stream_putw(s, 25); /* prefix sid sub-tlv len */
+			stream_putc(s, 0);  /* reserved */
+			stream_putc(s, BGP_PREFIX_SID_SRV6_L3_SERVICE_SID_INFO);
+			stream_putw(s, 21);     /* srv6 service sub-tlv len */
```

Comment:
Shouldn't this be BGP_PREFIX_SID_SRV6_L3_SERVICE_LENGTH?

Reply:
Yes. I'll fix